WordPress/Classes/ViewRelated/Comments/CommentsViewController.m

```diff
@@ -246,4 +245,33 @@ - (void)controllerDidChangeContent:(NSFetchedResultsController *)controller
         self.currentIndexPath = nil;
     }
 }
+
+#pragma mark - Syncs methods
+
+- (NSDate *)lastSyncDate
+{
+    return self.blog.lastCommentsSync;
+}
+
+- (BOOL)hasMoreContent {
+    return self.moreCommentsAvailable;
+}
+
+- (void)loadMoreWithSuccess:(void (^)())success failure:(void (^)(NSError *))failure {
+    self.blog.isSyncingComments = YES;
```

Comment:
The point of `isSyncingComment` is to be a (lousy) mutex for syncing. If you're not checking its value before calling sync, there's no point in setting it.